frontend/src/theme/darkTheme/themeColors.ts

```diff
@@ -40,4 +40,11 @@ export default {
   blue: colors.blue80,
   green: colors.green80,
   yellow: colors.yellow80,
+  // colors with md prefix, are used when coloring markdown
```

Comment:
Yeah, I forgot about Custom theme, so instead of defining mdColors in lightTheme, now lightTheme colors are defined in baseTheme: _frontend/src/theme/baseTheme/themeColors.ts_ 👍